HealthcareSystem/GraphicalEditor/Models/MapObjectRelated/MapObjectDoor.cs

```diff
@@ -0,0 +1,114 @@
+﻿using GraphicalEditor.Enumerations;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Controls;
+using System.Windows.Media;
+using System.Windows.Shapes;
+
+namespace GraphicalEditor.Models.MapObjectRelated
+{
+    public class MapObjectDoor
+    {
+        private Rectangle _rectangle;
+        private MapObjectMetrics _mapObjectMetrics;
+        private MapObjectDoorOrientation _mapObjectDoorOrientation;
+        private double _XShift;
+        private double _YShift;
+
```

Comment:
What are these fields used for? Maybe give a more descriptive name?

Reply:
By default, a door is positioned in the center of the border/wall defined by MapObjectDoorOrientation (top, bottom, left, right). If you would like to shift the door by X or Y, you could use those variables to do so.

Reply:
I cannot think of any better name. I could add a comment above them since that is the best way to understand the real usage.

Reply:
### Comment

Maybe like this:

```sh
        private double _XShiftFromCenter;
        private double _YShiftFromCenter;
```

### Proposal

Try to avoid comment, better is to put a long descriptive name.